src/site/content/en/blog/webtransport/index.md

```diff
@@ -177,15 +162,15 @@ Chrome does not [currently](https://bugs.chromium.org/p/chromium/issues/detail?i
 
 ### Streams APIs  {: #stream }
 
-Once you've connected to the server, you could also use QuicTransport to send and receive data via its Streams APIs.
+Once you've connected to the server, you could also use WebTransport to send and receive data via its Streams APIs.
 
 Each chunk of all streams is a `Uint8Array`. Unlike with the Datagram APIs, these streams are reliable. But each stream is independent, so data order across streams is not guaranteed.
 
 #### SendStream
 
-A <code>[SendStream](https://wicg.github.io/web-transport/#sendstream)</code> is created by the web client using the <code>createSendStream()</code> method of a `QuicTransport` instance, which returns a promise for the <code>SendStream</code>.
+A <code>[SendStream](https://wicg.github.io/web-transport/#sendstream)</code> is created by the web client using the <code>createSendStream()</code> method of a `WebTransport` instance, which returns a promise for the <code>SendStream</code>.
```

Comment:
ProTip: you can write that as:

```
A [`SendStream`](https://wicg.github.io/web-transport/#sendstream)</code> is created...
```

That'll respect the code blocks and you don't have to use the `<code>` element wrapper.

Reply:
That syntax was auto-generated by a Google Docs ➡️  Markdown export tool, so it was easier to just leave it as-is. 😄

Reply:
You should check the preview. I Don't think markdown inside tags will be rendered. I think it will be escaped.

Reply:
It looks fine to me: https://deploy-preview-5029--web-dev-staging.netlify.app/webtransport/#sendstream